src/normalizer.ts

```diff
@@ -54,7 +54,8 @@ export type InternalText = ObjectBase & {
 };
 export type InternalImage = ObjectBase & {
   kind: "image";
-  url: string;
+  url?: string;
+  src?: InternalImageSrc;
```

Comment:
Now I'm actually wondering why is this attribute optional at all 🤔

Reply:
Haha, you're right, it shouldn't be optional at all at this point.